.github/workflows/CI_Python.yml

```diff
@@ -23,3 +23,5 @@ jobs:
         run: |
           pip install .
           python -c "from xcdf import __version__; print(f'XCDF python bindings v{__version__}')"
+          pip install pytest
```

Comment:
do `pip install ".[tests]"` above, then you can remove this line and it's future proof if you add more `tests` dependencies

Reply:
right - I forgot about the `setup.cfg`...

Reply:
done